src/app/domain/backup/queries/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Q, QueryDefinition } from 'cozy-client'
+
+const DOCTYPE_FILES = 'io.cozy.files'
+
+export const buildFilesQuery = (dirId: string): QueryDefinition => {
+  return Q(DOCTYPE_FILES)
+    .where({
+      dir_id: dirId,
+      type: 'file'
+    })
+    .indexFields(['dir_id', 'type'])
+}
```

Comment:
Files is a pretty big doctype. You should `.select()` only the fields you want 🙏